ui/app/templates/components/secret-create-or-update.hbs

```diff
@@ -116,14 +116,19 @@
     {{/if}}
 
     <div class="field is-grouped box is-fullwidth is-bottomless">
-      <div class="control">
+      <Hds::ButtonSet>
         <Hds::Button @text="Save" type="submit" disabled={{@buttonDisabled}} data-test-secret-save={{true}} />
-      </div>
-      <div class="control">
-        <SecretLink @mode="list" @secret={{@model.parentKey}} class="button">
-          Cancel
-        </SecretLink>
-      </div>
+        {{#if @model.path}}
+          <Hds::Button
+            @text="Cancel"
+            @color="secondary"
+            @route="vault.cluster.secrets.backend.list"
+            @model={{@model.path}}
```

Comment:
Purposefully changed to `path` from `parentKey` so it's brought back to the right list view (previously it was going too far up the tree

Reply:
## fix 

> on `Cancel` form should go back to the `one/two` list view

![fix](https://github.com/hashicorp/vault/assets/68122737/5fd20f68-41c4-4c03-be77-82c99d2d4dad)

## bug

> previously was going back two levels, notice on `Cancel` we go back to `one/` even though we were creating a new secret from `one/two/`

![bug](https://github.com/hashicorp/vault/assets/68122737/a54465f1-8b02-4e6f-9351-a6e63f210f96)